modules/rbac-permission-sets.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control.adoc
+:_module-type: CONCEPT
+[id="rbac-permission-sets_{context}"]
+= System permission sets
+
+[role="_abstract"]
+{product-title} includes some default system permission sets that you can apply on users.
+You can also create custom permission sets as required.
+
+[cols="1,3"]
+|===
+| Permission set | Description
+
+| *Admin*
+| Provides read and write access to all resources.
+
+| *Analyst*
+| Provides read-only access for all resources.
+
+| *Continuous Integration*
+| This permission set is targeted for CI (Continuous Integration) systems and it includes the permissions required to enforce deployment policies.
+
+| *None*
+| No read and write permissions for any resource.
```

Comment:
This one is missing a verb so for parallelism, so maybe something like, "No read and write permissions are allowed for any resource."